src/logic/helpers/validateSource.js

```diff
@@ -0,0 +1,18 @@
+import retryingFailedImports from '../../util/retryingFailedImports';
+import {validatedSource} from '../../actions/errors';
+
+function importValidations() {
+  return retryingFailedImports(() =>
+    import(
+      /* webpackChunkName: "mainAsync" */
+      '../../validations'
+    ),
+  );
+}
+
+export default async ({language, source, projectAttributes}, dispatch) => {
```

Comment:
_Extreme_ nit / not a blocker, but all else being equal I’d make this a regular named function rather than an arrow function (since it’s at the top level of the module / not being inlined in a larger expression).